bool collision
float64 time_elapsed
bool new_run
bool end_of_action
float64 action_progress
int64 current_action
bool path_planner_terminate
bool lane_switch_failure_terminate
